typings/asobo-vcockpits-instruments/html_ui/Pages/VCockpit/Instruments/Shared/FlightElements/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Runway';
```

Comment:
We are not using the Asobo typings anymore. These classes now live in our tree and have been extended with new nav data fields added in various sim updates. The types in `@fmgc/types/fstypes/FSTypes` match what we're using (code is in the old `html_ui` tree.

Reply:
I'm not sure what you're asking me to do. I've only made the files that are there work without errors 😀.

Reply:
Well, you replaced  [this one](https://github.com/flybywiresim/a32nx/pull/6238/commits/2dbfd1e0079545ebd3eb930609272d2e9458b965) with the original Asobo definition that doesn't match our implementation. 😄

Reply:
That line imports a globally declared type defined in a TS definition file. That is not possible. By removing the import line the editor still assumes `OneWayRunway` declared in `src\fmgc\src\types\fstypes\FSTypes.d.ts` is used, but does not show any error.

![image](https://user-images.githubusercontent.com/1074786/143426868-3e65694c-a600-4428-a170-ba900d1b1fe0.png)